EKF/terrain_estimator.cpp

```diff
@@ -281,6 +283,8 @@ bool Ekf::isTerrainEstimateValid() const
 
 void Ekf::updateTerrainValidity()
 {
+	const bool rng_selected = _params.terrain_fusion_mode & TerrainFusionMask::TerrainFuseRangeFinder;
```

Comment:
Can we put `params.terrain_fusion_mode & TerrainFusionMask::TerrainFuseRangeFinder` in a function and use it in all location. Name suggestion `shouldUseRangeFinderForHagl` and `shouldUseOpticalFlowForHagl`